cli/internal/terraform/terraform/constellation-cluster/install-constellation.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+VERSION="latest"
+if ! command -v constellation &> /dev/null; then
+  OS=$(uname -s)
+  ARCH=$(uname -m)
+
+  URL=""
+
+  if [[ $OS == "Darwin" ]]; then
+    if [[ $ARCH == "arm64" ]]; then
+      URL="https://github.com/edgelesssys/constellation/releases/${VERSION}/download/constellation-darwin-arm64"
+    elif [[ $ARCH == "x86_64" ]]; then
+      URL="https://github.com/edgelesssys/constellation/releases/${VERSION}/download/constellation-darwin-amd64"
+    fi
+  elif [[ $OS == "Linux" ]]; then
+    if [[ $ARCH == "x86_64" ]]; then
+      URL="https://github.com/edgelesssys/constellation/releases/${VERSION}/download/constellation-linux-amd64"
+    elif [[ $ARCH == "arm64" ]]; then
+      URL="https://github.com/edgelesssys/constellation/releases/${VERSION}/download/constellation-linux-arm64"
+    fi
+  fi
+
+  if [ -z "$URL" ]; then
+    echo "Unsupported OS or architecture"
+    exit 1
+  else
+    curl -o constellation -LO $URL
+    sudo install constellation /usr/local/bin/constellation
```

Comment:
We should rather use the local file here. Installing the file to the user's machine might be an unwanted / unexpected side-effect of applying our module.